book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ Required ] Good job adding the `add_rental` method, however, as mentioned by the previous reviewer this method should be implemented differently. The `person` renting the book, and the rental `date` should be passed to the `add_rental` method as parameters. The `book` itself should be sent as a parameter to create the new rental achieving the `has-many` association. Please see the example below and rectify this.

## Example code
```ruby
  def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```